### ISSUE TITLE
2025-01-07, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#44]
# Today's Arxiv Papers

(1) [BoostStep: Boosting mathematical capability of Large Language Models via improved single-step reasoning](http://arxiv.org/abs/2501.03226v1), 2025-01-06

(2) [Dispider: Enabling Video LLMs with Active Real-Time Interaction via Disentangled Perception, Decision, and Reaction](http://arxiv.org/abs/2501.03218v1), 2025-01-06

(3) [Leveraging Explainable AI for LLM Text Attribution: Differentiating Human-Written and Multiple LLMs-Generated Text](http://arxiv.org/abs/2501.03212v1), 2025-01-06

(4) [Detecting AI-Generated Text in Educational Content: Leveraging Machine Learning and Explainable AI for Academic Integrity](http://arxiv.org/abs/2501.03203v1), 2025-01-06

(5) [The FACTS Grounding Leaderboard: Benchmarking LLMs' Ability to Ground Responses to Long-Form Input](http://arxiv.org/abs/2501.03200v1), 2025-01-06

(6) [CLIX: Cross-Lingual Explanations of Idiomatic Expressions](http://arxiv.org/abs/2501.03191v1), 2025-01-06

(7) [Semantic Captioning: Benchmark Dataset and Graph-Aware Few-Shot In-Context Learning for SQL2Text](http://arxiv.org/abs/2501.03166v1), 2025-01-06

(8) [Large language models for artificial general intelligence (AGI): A survey of foundational principles and approaches](http://arxiv.org/abs/2501.03151v1), 2025-01-06

(9) [VicSim: Enhancing Victim Simulation with Emotional and Linguistic Fidelity](http://arxiv.org/abs/2501.03139v1), 2025-01-06

(10) [PRMBench: A Fine-grained and Challenging Benchmark for Process-Level Reward Models](http://arxiv.org/abs/2501.03124v1), 2025-01-06

(11) [CAT: Content-Adaptive Image Tokenization](http://arxiv.org/abs/2501.03120v1), 2025-01-06

(12) [LangFair: A Python Package for Assessing Bias and Fairness in Large Language Model Use Cases](http://arxiv.org/abs/2501.03112v1), 2025-01-06

(13) [Retrieval-Augmented TLAPS Proof Generation with Large Language Models](http://arxiv.org/abs/2501.03073v1), 2025-01-06

(14) [Design and implementation of tools to build an ontology of Security Requirements for Internet of Medical Things](http://arxiv.org/abs/2501.03067v1), 2025-01-06

(15) [ChronoSense: Exploring Temporal Understanding in Large Language Models with Time Intervals of Events](http://arxiv.org/abs/2501.03040v1), 2025-01-06

(16) [Quantization Meets Reasoning: Exploring LLM Low-Bit Quantization Degradation for Mathematical Reasoning](http://arxiv.org/abs/2501.03035v1), 2025-01-06

(17) [Analyzing Fine-tuning Representation Shift for Multimodal LLMs Steering alignment](http://arxiv.org/abs/2501.03012v1), 2025-01-06

(18) [CALM: Curiosity-Driven Auditing for Large Language Models](http://arxiv.org/abs/2501.02997v1), 2025-01-06

(19) [Registering Source Tokens to Target Language Spaces in Multilingual Neural Machine Translation](http://arxiv.org/abs/2501.02979v1), 2025-01-06

(20) [FlipedRAG: Black-Box Opinion Manipulation Attacks to Retrieval-Augmented Generation of Large Language Models](http://arxiv.org/abs/2501.02968v1), 2025-01-06

(21) [Socratic Questioning: Learn to Self-guide Multimodal Reasoning in the Wild](http://arxiv.org/abs/2501.02964v1), 2025-01-06

(22) [SceneVTG++: Controllable Multilingual Visual Text Generation in the Wild](http://arxiv.org/abs/2501.02962v1), 2025-01-06

(23) [MotionBench: Benchmarking and Improving Fine-grained Video Motion Understanding for Vision Language Models](http://arxiv.org/abs/2501.02955v1), 2025-01-06

(24) [DeCon: Detecting Incorrect Assertions via Postconditions Generated by a Large Language Model](http://arxiv.org/abs/2501.02901v1), 2025-01-06

(25) [MDP3: A Training-free Approach for List-wise Frame Selection in Video-LLMs](http://arxiv.org/abs/2501.02885v1), 2025-01-06

(26) [IIMedGPT: Promoting Large Language Model Capabilities of Medical Tasks by Efficient Human Preference Alignment](http://arxiv.org/abs/2501.02869v1), 2025-01-06

(27) [Large Language Models for Video Surveillance Applications](http://arxiv.org/abs/2501.02850v1), 2025-01-06

(28) [Graph-based Retrieval Augmented Generation for Dynamic Few-shot Text Classification](http://arxiv.org/abs/2501.02844v1), 2025-01-06

(29) [Randomly Sampled Language Reasoning Problems Reveal Limits of LLMs](http://arxiv.org/abs/2501.02825v1), 2025-01-06

(30) [InfiFusion: A Unified Framework for Enhanced Cross-Model Reasoning via LLM Fusion](http://arxiv.org/abs/2501.02795v1), 2025-01-06

(31) [GeAR: Generation Augmented Retrieval](http://arxiv.org/abs/2501.02772v1), 2025-01-06

(32) [Visual Large Language Models for Generalized and Specialized Applications](http://arxiv.org/abs/2501.02765v1), 2025-01-06

(33) [Ultrasound-QBench: Can LLMs Aid in Quality Assessment of Ultrasound Imaging?](http://arxiv.org/abs/2501.02751v1), 2025-01-06

(34) [TARDiS : Text Augmentation for Refining Diversity and Separability](http://arxiv.org/abs/2501.02739v1), 2025-01-06

(35) [Artificial Intelligence in Creative Industries: Advances Prior to 2025](http://arxiv.org/abs/2501.02725v1), 2025-01-06

(36) [KG-CF: Knowledge Graph Completion with Context Filtering under the Guidance of Large Language Models](http://arxiv.org/abs/2501.02711v1), 2025-01-06

(37) [QuIM-RAG: Advancing Retrieval-Augmented Generation with Inverted Question Matching for Enhanced QA Performance](http://dx.doi.org/10.1109/ACCESS.2024.3513155), 2025-01-06

(38) [EAGLE: Enhanced Visual Grounding Minimizes Hallucinations in Instructional Multimodal Models](http://arxiv.org/abs/2501.02699v1), 2025-01-06

(39) [Decoding specialised feature neurons in LLMs with the final projection layer](http://arxiv.org/abs/2501.02688v1), 2025-01-05

(40) [Layer-Level Self-Exposure and Patch: Affirmative Token Mitigation for Jailbreak Attack Defense](http://arxiv.org/abs/2501.02629v1), 2025-01-05

(41) [Cracks in The Stack: Hidden Vulnerabilities and Licensing Risks in LLM Pre-Training Datasets](http://arxiv.org/abs/2501.02628v1), 2025-01-05

(42) [HALO: Hadamard-Assisted Lossless Optimization for Efficient Low-Precision LLM Training and Fine-Tuning](http://arxiv.org/abs/2501.02625v1), 2025-01-05

(43) [LLMs Help Alleviate the Cross-Subject Variability in Brain Signal and Language Alignment](http://arxiv.org/abs/2501.02621v1), 2025-01-05

(44) [TAPAS: Thermal- and Power-Aware Scheduling for LLM Inference in Cloud Platforms](http://arxiv.org/abs/2501.02600v1), 2025-01-05

(45) [LeetDecoding: A PyTorch Library for Exponentially Decaying Causal Linear Attention with CUDA Implementations](http://arxiv.org/abs/2501.02573v1), 2025-01-05

(46) [Multi-LLM Collaborative Caption Generation in Scientific Documents](http://arxiv.org/abs/2501.02552v1), 2025-01-05

(47) [Transformers Simulate MLE for Sequence Generation in Bayesian Networks](http://arxiv.org/abs/2501.02547v1), 2025-01-05

(48) [Evaluating Large Language Models Against Human Annotators in Latent Content Analysis: Sentiment, Political Leaning, Emotional Intensity, and Sarcasm](http://arxiv.org/abs/2501.02532v1), 2025-01-05

(49) [Towards New Benchmark for AI Alignment & Sentiment Analysis in Socially Important Issues: A Comparative Study of Human and LLMs in the Context of AGI](http://arxiv.org/abs/2501.02531v1), 2025-01-05

(50) [Vision-Driven Prompt Optimization for Large Language Models in Multimodal Generative Tasks](http://arxiv.org/abs/2501.02527v1), 2025-01-05

(51) [CHAIR-Classifier of Hallucination as Improver](http://arxiv.org/abs/2501.02518v1), 2025-01-05

(52) [ToolHop: A Query-Driven Benchmark for Evaluating Large Language Models in Multi-Hop Tool Use](http://arxiv.org/abs/2501.02506v1), 2025-01-05

(53) [LLMPC: Large Language Model Predictive Control](http://arxiv.org/abs/2501.02486v1), 2025-01-05

(54) [Decoding News Bias: Multi Bias Detection in News Articles](http://arxiv.org/abs/2501.02482v1), 2025-01-05

(55) [Hengqin-RA-v1: Advanced Large Language Model for Diagnosis and Treatment of Rheumatoid Arthritis with Dataset based Traditional Chinese Medicine](http://arxiv.org/abs/2501.02471v1), 2025-01-05

(56) [Towards Omni-RAG: Comprehensive Retrieval-Augmented Generation for Large Language Models in Medical Applications](http://arxiv.org/abs/2501.02460v1), 2025-01-05

(57) [Understand, Solve and Translate: Bridging the Multilingual Mathematical Reasoning Gap](http://arxiv.org/abs/2501.02448v1), 2025-01-05

(58) [RTLMarker: Protecting LLM-Generated RTL Copyright via a Hardware Watermarking Framework](http://arxiv.org/abs/2501.02446v1), 2025-01-05

(59) [A Statistical Hypothesis Testing Framework for Data Misappropriation Detection in Large Language Models](http://arxiv.org/abs/2501.02441v1), 2025-01-05

(60) [Efficient Deployment of Large Language Models on Resource-constrained Devices](http://arxiv.org/abs/2501.02438v1), 2025-01-05

(61) [FOLDER: Accelerating Multi-modal Large Language Models with Enhanced Performance](http://arxiv.org/abs/2501.02430v1), 2025-01-05

(62) [Scaling Laws for Floating Point Quantization Training](http://arxiv.org/abs/2501.02423v1), 2025-01-05

(63) [GenTREC: The First Test Collection Generated by Large Language Models for Evaluating Information Retrieval Systems](http://arxiv.org/abs/2501.02408v1), 2025-01-05
